task_02/xieyu_wang/graphic.cpp

```diff
@@ -0,0 +1,89 @@
+#include <array>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+constexpr int MAX_VEC = 18;
+constexpr int MAX_PTS = 13;
+struct Edge {
+  int from;
+  int edge_id;
+  int to;
+};
+static constexpr std::array<Edge, MAX_VEC> netWork = {{{0, 0, 1},
+                                                       {0, 1, 2},
+                                                       {0, 2, 3},
+                                                       {1, 3, 4},
+                                                       {1, 4, 5},
+                                                       {1, 5, 6},
+                                                       {2, 6, 7},
+                                                       {2, 7, 8},
+                                                       {3, 8, 9},
+                                                       {4, 9, 10},
+                                                       {5, 10, 10},
+                                                       {6, 11, 10},
+                                                       {7, 12, 11},
+                                                       {8, 13, 11},
+                                                       {9, 14, 12},
+                                                       {10, 15, 13},
+                                                       {11, 16, 13},
+                                                       {12, 17, 13}}};
+bool WriteToFile() {
```

Comment:
Read `Naming Convention`, it is written there how to name classes, structures, variables and functions

Reply:
Separate the logic of preparing `std::string` and working with file into different functions. It is always better to keep function as small as possible, and to encapsulate as little logic as possible into them.

Reply:
`bool` is not a good way of handling errors. You need to try to be more expressive with your language. Just simple `bool` doesn't give us much information. You can do it in different ways, for example you use `Enum Class`
```suggestion
enum class WriteToFileStatus { Ok, FileIsNotOpened };

WriteToFileStatus writeToFile() {
```